composer.json

```diff
@@ -52,6 +52,7 @@
     "lint": "./vendor/bin/phpcs --standard=phpcs.xml .",
     "fix": "./vendor/bin/phpcbf --standard=phpcs.xml .",
     "test": "../../../vendor/bin/phpunit -c phpunit.xml.dist --colors=always",
-    "release": "bin/release"
+    "release": "bin/release",
+    "eslint": "bin/eslint.sh"
```

Comment:
What package json file? I think are got this confused with the noUiSlider where we created the package json file.

Reply:
Please see [this article by Shopware](https://www.shopware.com/en/news/installing-your-own-dependencies-via-npm/), which explains how to add npm dependencies. In our case we need eslint as a dev package for storefront and for administration.
As you already mentioned, we already use noUiSlider for storefront in another story, so it probably makes sense to build on that branch to avoid merge conflicts.

Reply:
It was not needed, as it would have required a whole lot of setup and several dependencies which would have been redundant in my opinion. Have fixed it and the build is now failing due to only lint issues.